op-challenger/fault/game.go

```diff
@@ -0,0 +1,13 @@
+package fault
+
+// Game is an interface that represents the state of a dispute game.
```

Comment:
If all we have is the root claim & get parent, how do we iterate the state?

`Put` is good, but instead of `GetParent` + `RootClaim` we need `ClaimParis() []{claim: Claim, parent: Claim}`